public/app/plugins/datasource/mixed/plugin.json

```diff
@@ -13,7 +13,8 @@
     "logos": {
       "small": "img/icn-mixeddatasources.svg",
       "large": "img/icn-mixeddatasources.svg"
-    }
+    },
+    "links": [{ "name": "Raise issue", "url": "https://github.com/grafana/grafana/issues/new" }]
```

Comment:
docs could be linked as https://grafana.com/docs/grafana/latest/datasources/#special-data-sources